messages/en.json

```diff
@@ -0,0 +1,4 @@
+{
+  "buyer-organizations-supplier.admin.navigation.label-group": "Organizations",
+  "buyer-organizations-supplier.admin.navigation.label-main": "Buyer Organizations"
```

Comment:
I couldn't find "buyer-organizations-supplier.admin.navigation.label-main" being used somewhere else.
Is it necessary?